libs/ui/src/pure/Tooltip/index.tsx

```diff
@@ -6,6 +6,8 @@ import Popover, { PopoverProps } from '../Popover'
 
 import { Command } from '@cowprotocol/types'
 
+const TOOLTIP_CLOSE_DELAY = 300
```

Comment:
Nitpick: add a comment as to what units this is supposed to be

```suggestion
const TOOLTIP_CLOSE_DELAY = 300 // in ms
```

Reply:
Well, I modeled the stickiness and the delayed closing. I could kill the click for stickiness after this PR adds little value, but its still kind of nice...

Now that I think more, I think I will kill it, this way I can ger rid of the scroll listener and the bug @elena-zh is reporting :)

Reply:
Will fix in the PR that supersedes this one https://github.com/cowprotocol/cowswap/pull/4339